VirtoCommerce.CatalogPersonalizationModule.Core/Model/EffectiveTag.cs

```diff
@@ -0,0 +1,8 @@
+namespace VirtoCommerce.CatalogPersonalizationModule.Core.Model
+{
+    public class EffectiveTag
```

Comment:
Please make a two static fabric  methods
EffectiveTag  InheritedTag(tag)
EffectiveTag NonInheritedTag(tag)

and use them for the instances creation

Reply:
done